nvflare/app_common/workflows/cross_site_model_eval.py

```diff
@@ -434,22 +441,37 @@ def _accept_val_result(self, client_name: str, result: Shareable, fl_ctx: FLCont
 
             self._val_results[client_name][model_owner] = {}
```

Comment:
This line is also problematic - the client_name may not exists in self._val_results yet.